selfdrive/manager.py

```diff
@@ -406,7 +406,7 @@ def send_managed_process_signal(name, sig):
 
 def manager_init(should_register=True):
   if should_register:
-    reg_res = register()
+    reg_res = register(spinner)
```

Comment:
can't we just set the spinner here instead of passing it in? should be fast enough to not flicker if it doesn't need to register

Reply:
You definitely see it. At least on tici